packages/vite/src/utils/warmup.ts

```diff
@@ -34,6 +34,10 @@ function isBuiltin (id: string) {
   return id.startsWith('node:') || builtins.has(id)
 }
 
+function isVirtual (id: string) {
+  return id.startsWith('virtual:')
```

Comment:
`virtual:` is a community conversion, also it could be `\0` or honestly everything. I wonder if there could be a more general solution to this (I am ok with treating them a bit specifically at this moment, but it might not be a solid solution for the long term)